subprojects/core-api/src/main/java/org/gradle/api/artifacts/dsl/RepositoryHandler.java

```diff
@@ -137,7 +139,9 @@
      *
      * @return the added resolver
      * @see #jcenter(Action)
+     * @deprecated Use {@link #mavenCentral()} instead
```

Comment:
I think we should also mention something about the why + link to the JCenter sunset blog post in the javadoc for both of jcenter methods.

Reply:
Fixed.